controllers/topics.js

```diff
@@ -0,0 +1,14 @@
+const { selectAllTopics } = require('../models/topics');
+
+exports.getAllTopics = (req, res, next) =>
+{
+    selectAllTopics()
+    .then((topics) =>
+    {
+        res.status(200).json(topics);
```

Comment:
This should be returned on an object - such as {topics: topics}. This is how we can communicate to other users of our code what it is that we are sending.

Reply:
This should also be sent on res.status(200).send({whatever you want to send}), not on .json, please! Take another look at the express notes for guidance please!